modules/gdscript/gdscript_function.cpp

```diff
@@ -269,6 +269,7 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 	r_err.error = Variant::CallError::CALL_OK;
 
 	Variant self;
+	Variant static_ref;
```

Comment:
It seems this is not being set. If you cherry-picked my commit directly you also have to pick the fixup: https://github.com/godotengine/godot/commit/0f1da724924fc494bfd316c57757e0e1d7b90143

Reply:
Alright. Done!